fix(packages/sui-studio): get commits list from github api (#1085)
SUI-Components/sui: packages/sui-studio/README.md

```

```bash

> # run the build just for changed components in a branch and also execute whatever you want before the build (phoenix, lint...)

Reviewer: ```suggestion
# run the build just for changed components on a PR and also execute whatever you want before the build (phoenix, lint...)
```
